docs/core/pipes/date-time.pipe.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: Date Time Pipe
+Added: v1.0.0
+Status: Active
+Last reviewed: 2023-08-30
+---
+
+# [Date Time Pipe](../../../lib/pipes/date-time.pipe.ts "Defined in date-time.pipe.ts")
```

Comment:
is that path proper? If you click at that then redirection does not work so looks like path is incorrect.

Reply:
corrected the path